clang/lib/CodeGen/CodeGenModule.cpp

```diff
@@ -2047,9 +2047,9 @@ void CodeGenModule::EmitCtorList(CtorList &Fns, const char *GlobalName) {
   llvm::Type *CtorPFTy = llvm::PointerType::get(CtorFTy,
       TheModule.getDataLayout().getProgramAddressSpace());
 
-  // Get the type of a ctor entry, { i32, void ()*, i8* }.
```

Comment:
Probably should handle the ctor / dtor case separately from the used one, since that may be more meaningful

Reply:
Will do, thanks!

Reply:
This is still one PR that touches both?

Reply:
Apologies, took slightly longer than planned, please see #93914.